### DISCUSSION TITLE
Allow configuring timezone in datetime display

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#4730] [Feature Requests]
I have formatted the date and time as:

`yyyy-MM-dd HH:mm:ss`

![image](https://user-images.githubusercontent.com/8030134/112838908-29928e80-90d0-11eb-8adf-dbeea3055d7e.png)

2021-03-29 01:11:23

![image](https://user-images.githubusercontent.com/8030134/112838787-0962cf80-90d0-11eb-97fa-27f6cc9c2853.png)

But I don’t know how to make it go forward for 8 hours,like:

2021-03-29 `01`:11:23 + 8 = 2021-03-29 `09`:11:23

In v8, it adds 8 hours by default.

**Comment by rijkvanzanten** [comment]
There's currently no way to configure the timezone used for the datetime display. I'll consider this discussion a feature request for that 👍🏻

**Reply by cdwmhcc** [reply]
Haha,OK,I thought at first that I was missing something, so I checked the documentation carefully, but found nothing.

**Reply by badrange** [reply]
Consider this a feature request from our team too. Ideally it would be possible to choose between showing the display to use a server configured timezone OR timezone specified by the user themselves.

This would only work for timestamp fields I presume.

**Reply by sandyksit** [reply]
> There's currently no way to configure the timezone used for the datetime display. I'll consider this discussion a feature request for that 👍🏻

Is this available in new versions?

**Reply by eikaramba** [reply]
i believe no

**Comment by eikaramba** [comment]
The [latest PR](https://github.com/directus/directus/pull/10956) changed everything to UTC timezone, which solved a lot of issues. Now one idea can be to integrate a timezone setting for each user and use that to show the local time for the current logged in user if that is activated.

todos:
- add `timezone` setting (string) to user entity
- add `showLocalTime` setting (boolean) to datetime/timestamp fields
- if `showLocalTime` true for this field
  - transform `utc->localtime` on display.
  - transform `localtime -> utc` upon updating value.

The api and persistent db values should be not affected by this. this is merely a frontend change for the admin app

Just my thoughts about this as i am also in favor to support this. i am even thinking about building an extension for this or doing a PR, but don't expect anything soon. :)

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by pbischoffcg** [comment]
Hello everyone,

I wrote a feature request, that handles this topic. If you are interested in this feature you can upvote the feature request [here](https://github.com/directus/directus/discussions/19460) and increase the probability that this feature comes to directus.
